Assignment/solution_student/index.html

```diff
@@ -1,33 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <title>FEWD HTML Template</title>
-    <meta name="description" content="An HTML Starter Template">
+    <title>Relaxr</title>
+    <meta name="description" content="Relaxr website">
     
-    <link rel="stylesheet" href="css/normalize.css"><!-- CSS reset: http://necolas.github.io/normalize.css/ -->
-    <link rel="stylesheet" href="css/style.css"><!-- Our site's stylesheet -->
-
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,600italic,700,700italic,800,800italic' rel='stylesheet' type='text/css'>
     <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
   </head>
+
+
   <body>
+    <header>
 
-    <!-- YOUR MARKUP GOES HERE! -->
-    <div class="container">
-    
-      <header>
-        <h1>FEWD HTML Template</h1>
-      </header>
+          <h1>Relaxr</h1>
+          <h2>Get piece of mind with a single tap</h2>
+          <a id="now" href="#">Get it Now</a>
+      
+    </header>
+
+
+    <div class="main">
+        <h3>Benefits</h3>
+        <p>The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+
+        <ul>
+          <li>Schedule meetings for you</li>
+          <li>Excel automation to complete your work for you</li>
+          <li>Responds to emails on your behalf</li>
+          <li>Does all your work for you with our revolutionary AI technology</li>
+        </ul>
+      </div>
+
+
+      <blockquote class="testimonial">
+          “Relaxr changed my life. I’ve been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work.” <br><span>- Amanda, Intuit
```

Comment:
oops — looks like you forgot to close your `<span>` here! Also, remember, avoid using `<br>` to space things out. This is on the border of being a legitimate use case, since you really do need to break onto the next line, but you could also do it in css:
```
.testimonial span {
  display: block;
}
```
That will put the span on a new line just like a block element would be.